app/models/user.rb

```diff
@@ -1,5 +1,27 @@
 class User < ApplicationRecord
+  include UsersHelper
   devise :database_authenticatable
   has_many :interests, dependent: :delete_all
   validates :email, uniqueness: true
+
+  def self.to_csv
+    attributes = %w[id email gender age interests_list]
+
+    CSV.generate(headers: true) do |csv|
+      csv << attributes
+      all.each { |user| csv << attributes.map { |attr| user.send(attr) } }
```

Comment:
Use `public_send` over `send`, as the latter one may confuse someone into thinking that you're calling a private method.